compiler/noirc_frontend/src/monomorphization/ast.rs

```diff
@@ -349,18 +364,18 @@ impl Program {
     }
 
     pub fn main_id() -> FuncId {
-        FuncId(0)
+        FuncId::Interned(0)
     }
 
     pub fn take_main_body(&mut self) -> Expression {
-        self.take_function_body(FuncId(0))
+        self.take_function_body(FuncId::Dummy)
```

Comment:
```suggestion
        self.take_function_body(self.main_id())
```
Not `0` here? Looks like this method is unused either way